grpcbp/prometheus.go

```diff
@@ -6,8 +6,7 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
 
-	//lint:ignore SA1019 This library is internal only, not actually deprecated
-	"github.com/reddit/baseplate.go/internalv2compat"
+	internal_prometheusbp "github.com/reddit/baseplate.go/internal/prometheusbp"
```

Comment:
The style established by the standard library is that you don't name anything with underscores, even package names.  Either `internalprometheusbp` or `prometheusbpint` or something like that.